tutorials/deploy-meilisearch-instance/index.mdx

```diff
@@ -0,0 +1,168 @@
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) (specifically **InstancesFullAccess**) allowing you to perform actions in the intended Organization
+- An [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/)
+- An [Instance](/compute/instances/how-to/create-an-instance/) with the [Docker InstantApp](/compute/instances/reference-content/images-and-instantapps/#instantapps) or with the [Docker Engine](https://docs.docker.com/engine/install/) installed
+
+
+## Connecting to your Instance and starting Meilisearch
+
+1. Open a terminal and connect to your Instance using [SSH](/compute/instances/how-to/connect-to-instance/). Replace `<INSTANCE_IP>` with your Instance's IP (Instances > Overview > Flexible IP > the first IP address that displays in the list).
+
+    ```
+    ssh root@<INSTANCE_IP>
+    ```
+
+    <Message type="tip">
+     If you are using an Instance with a Docker InstantApp, run `docker -v` to make sure that the Docker engine is already installed. The current docker engine version should display.
+    </Message>
+2. In the same terminal, generate a master key for Meilisearch. This key will secure your Meilisearch instance.
+
+    ```
+    export MEILI_MASTER_KEY=$(openssl rand -base64 16)
+    ```
+3. Run the following command to display the master key:
+
+    ```
+    echo $MEILI_MASTER_KEY
+    ```
+    <Message type="important">
+     Copy and save your master key as you will need it in the next steps.
+    </Message>
+4. Run the MeiliSearch container using Docker. This command sets up Meilisearch to store data persistently on the volume located at `$(pwd)/meili_data:/meili_data` on your Instance.
+
+    ```
+    docker run -d -p 7700:7700 -v $(pwd)/meili_data:/meili_data  -e $MEILI_MASTER_KEY getmeili/meilisearch:v1.9
+    ```
+5. Check that the container is running:
+    ```
+    docker ps
+    ```
+    You should see an output similar to the following:
+    ```
+    CONTAINER ID   IMAGE                       COMMAND                  CREATED          STATUS          PORTS                                       NAMES
+    0432254e4d64   getmeili/meilisearch:v1.9   "tini -- /bin/sh -c …"   56 minutes ago   Up 56 minutes   0.0.0.0:7700->7700/tcp, :::7700->7700/tcp   flamboyant_dewdney
+    ```
+    <Message type="tip">
+     You can stop the Meilisearch container using `docker container stop <container_name>`.
+    </Message>
+6. Use the following command to check that Meilisearch is running.
+
+    ```
+    curl -X GET 'http://localhost:7700/version' -H 'Authorization: Bearer '$MEILI_MASTER_KEY
+    ```
+
+    You should see an output similar to the following:
+    ```json
+    {"commitSha":"0df84bbba7db46680e83843dd6454257161a282f","commitDate":"2024-06-27T12:47root@scwrororororrrrrrrrrrrrrrrrorrrorrrrrrrrrrrrroot@scw-lucid-satoshi:~#
```

Comment:
This line feels like there is a copy paste issue.
The format should be json only, like {"commitSha":"0df84bbba7db46680e83843dd6454257161a282f","commitDate":"2024-06-27T12", "pkgVersion":aaaaaa}